nautobot/docs/administration/nautobot-server.md

```diff
@@ -284,24 +284,24 @@ This will run the following management commands with default settings, in order:
 - `invalidate all`
 
 !!! note
-    Commands listed here that are not covered in this document here are Django built-in commands. 
+    Commands listed here that are not covered in this document here are Django built-in commands.
 
-`--no-clearsessions`<br>
+`--no-clearsessions`
 Do not automatically clean out expired sessions.
 
-`--no-collectstatic`<br>
+`--no-collectstatic`
 Do not automatically collect static files into a single location.
 
-`--no-invalidate-all`<br>
+`--no-invalidate-all`
 Do not automatically invalidate cache for entire application.
 
-`--no-migrate`<br>
+`--no-migrate`
 Do not automatically perform any database migrations.
 
-`--no-remove-stale-contenttypes`<br>
+`--no-remove-stale-contenttypes`
 Do not automatically remove stale content types.
 
-`--no-trace-paths`<br>
+`--no-trace-paths`
 Do not automatically generate missing cable paths.
```

Comment:
I removed these HTML tags for the Markdown Linting MD033
https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md033

Reply:
If I can add to this, perhaps @matt852 you should add two spaces at the end of the line, this will do the same as the `<br>` and be allowed by Markdownlint.

Reply:
Fixed.  The double spaces made it look off as well.  I looked it up and learned that adding two whitespaces to the end of the line will fix the formatting to be like it is now, without needing the `<br />` tags.